Automattic/jetpack
# Sharing: improve the display of the official X button

Follow-up to #33134

## Proposed changes:

While there is "X" official sharing button, we should still have the nicest display we can have when folks pick that official style.


### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Social
* Enable sharing buttons
* Go to Settings > Sharing
* Enable the X sharing button
* Pick the official button style
    * Ensure the button looks good right there in wp-admin
* Save your changes
* View a post on the frontend.
    * Ensure the button looks good there too.
